bin/component-cli

```diff
@@ -8,7 +8,7 @@ import sys
 
 import yaml
 
-import gci.componentmodel as cm
+import ocm as cm
```

Comment:
Let us also get rid of the alias cm here, I think ocm is already short enough :-)

Reply:
yes, I agree. Let's see how well my `sed`-skills are